src/tools/ArtifactsUploader/ArtifactsUploader/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "profiles": {
+    "ArtifactsUploader": {
+      "commandName": "Project",
+      "commandLineArgs": "--container results --branch csfiles --artifacts . --searchPatterns *cs --storageUrl https://pvscmdupload.blob.core.windows.net --token \"?sv=2018-03-28&ss=bf&srt=sco&sp=rwdlac&se=2019-05-24T07:15:29Z&st=2019-05-23T23:15:29Z&spr=https&sig=IjS1XEZdEm0dfieRK3hccg22%2Bh%2BX%2FrtwMDFMdBqNPYo%3D\" --job testjob --skipCompression true"
```

Comment:
We should not put the token in this json file.

Reply:
great catch. I missed this file getting created. My bad. The token already expired but I am going to roll keys on this storage account anyway.

Reply:
I would add this file to `.gitignore`, so it does not happen in the future.

Reply:
I was going to but found some evidence (1, 2) that perhaps it's more integral than the old .user project files..

1: https://github.com/github/gitignore/pull/2705
2: https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-run?tabs=netcore21

Reply:
Optionally, there is a CI tool that scan the repo to catch these things (though it would be too late once it's on a PR).